### PR TITLE
Refactor Test For TestMakeBountyPayment To Use A Real Postgres DB For The Test

### PR DESCRIPTION
**Pull request by AbdulWahab3181**
## Describe your changes

image

## Issue ticket number and link

Closes https://github.com/stakwork/sphinx-tribes/issues/1855

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [x] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by saithsab877** [comment]
h

File: handlers/bounty_test.go [hunk 1]
```diff
@@ -1197,22 +1200,70 @@ func TestMakeBountyPayment(t *testing.T) {
...
+	bHandler := NewBountyHandler(mockHttpClient, db.TestDB)
 
 	var mutex sync.Mutex
 	var processingTimes []time.Time
 
-	bountyID := uint(1)
+	now := time.Now().Unix()
+	bountyOwnerId := "owner_pubkey"
+
+	db.TestDB.DeleteWorkspaceBudget()
```
Comment (elraphty) [inline handlers/bounty_test.go:1211]: @AbdulWahab3181 why do you have to delete the Workspace Budget, the budgets are different for different workspaces

**Comment by elraphty** [comment]
@AbdulWahab3181 you didn't assert for these:

- Assert that the bounty is Paid value is set to true
- Assert that the bounty amount is deducted from the Workspace budget

File: handlers/bounty_test.go (hunk 1 above)
Comment (AbdulWahab3181) [reply handlers/bounty_test.go:1211]: @elraphty I deleted for safe side. Should I remove?

Comment (elraphty) [reply handlers/bounty_test.go:1211]: Yeah remove it

**Comment by elraphty** [comment]
> @AbdulWahab3181 you didn't assert for these:
> 
> * Assert that the bounty is Paid value is set to true
> * Assert that the bounty amount is deducted from the Workspace budget

also address this

File: handlers/bounty_test.go (hunk 1 above)
Comment (AbdulWahab3181) [reply handlers/bounty_test.go]: @elraphty Addressed

**Comment by AbdulWahab3181** [comment]
> > @AbdulWahab3181 you didn't assert for these:
> > 
> > * Assert that the bounty is Paid value is set to true
> > * Assert that the bounty amount is deducted from the Workspace budget
> 
> also address this

@elraphty Addressed

**Pull request merged**